docs/migrating-to-tool-kit.md

```diff
@@ -75,17 +76,16 @@ Would you like to install any additional plugins?
 
 You are then given the opportunity to add various plugins for other tools you
 use in your project. It makes sense to choose all the ones you are already
-using, e.g., the `@dotcom-tool-kit/mocha` plugin if you're using Mocha for unit
-tests, but now is also a good chance to try out new tools you've been meaning to
-integrate into your stack, such as `dotcom-tool-kit/prettier`!
+using, but now is also a good chance to try out new tools you've been meaning to
+integrate into your stack, such as `dotcom-tool-kit/prettier`! Note: if you don't already have prettier installed it may be initially a little disruptive with many formatting changes.
```

Comment:
**nitpick**:

```suggestion
integrate into your stack, such as `dotcom-tool-kit/prettier`! Note: if you don't already have Prettier installed it may be initially a little disruptive with many formatting changes.
```